src/main/java/application/FormController.java

```diff
@@ -102,6 +111,7 @@ public SubmitResponseBean submitForm(@RequestBody SubmitRequestBean submitReques
         FormRecordProcessorHelper.processXML(formEntrySession.getSandbox(), formEntrySession.submitGetXml());
         SubmitResponseBean submitResponseBean = new SubmitResponseBean(formEntrySession);
         log.info("Submit response bean: " + submitResponseBean);
+        sessionRepo.delete(submitRequestBean.getSessionId());
```

Comment:
@benrudolph And I spoke about this, going to move submission to happen in Formplayer in a future PR